### PR TITLE
feat: publish es_ga result in event

### PR DESCRIPTION
**Pull request by lasrinnil**
## References

* #886

### CODE REVIEW

File: source/dotnet/Services/Application/IIntegrationEventPublisher.cs [hunk 1]
```diff
@@ -21,5 +21,7 @@ public interface IIntegrationEventPublisher
 {
     Task PublishAsync(ProcessCompletedEventDto processCompletedEvent);
 
-    Task PublishAsync(ProcessStepResult processStepResultDto, ProcessCompletedEventDto processCompletedEventDto);
+    Task PublishCalculationResultForTotalGridAreaAsync(ProcessStepResult processStepResultDto, ProcessCompletedEventDto processCompletedEventDto);
+
+    Task PublishCalculationResultForEnergySupplierAsync(ProcessStepResult processStepResultDto, ProcessCompletedEventDto processCompletedEventDto, string energySupplierGln);
```
Comment (BjarkeMeier) [inline source/dotnet/Services/Application/IIntegrationEventPublisher.cs:26]: Does this signature indicate too much responsibility? I would expect something more similar to:
```suggestion
    Task PublishAsync(IntegrationEvent integrationEvent);
```

Comment (lasrinnil) [reply source/dotnet/Services/Application/IIntegrationEventPublisher.cs:26]: Could you elaborate on your approach? Perhaps create an example of how you would expect `IntegrationEvent` to look.

Comment (lasrinnil) [reply source/dotnet/Services/Application/IIntegrationEventPublisher.cs:26]: A refactor of that size now would seem highly inefficient to me with the Outbox approach coming in a near feature.

Comment (BjarkeMeier) [reply source/dotnet/Services/Application/IIntegrationEventPublisher.cs:26]: Okay 😄

File: source/dotnet/Services/Application/Processes/ProcessApplicationService.cs [hunk 2]
```diff
@@ -51,11 +55,52 @@ public async Task PublishProcessCompletedIntegrationEventsAsync(ProcessCompleted
...
+    }
+
+    private async Task PublishCalculationResultCompletedForEnergySuppliersAsync(ProcessCompletedEventDto processCompletedEvent, TimeSeriesType timeSeriesType)
+    {
+            var actors = await _actorRepository.GetEnergySuppliersAsync(
+                processCompletedEvent.BatchId,
+                new GridAreaCode(processCompletedEvent.GridAreaCode),
+                timeSeriesType).ConfigureAwait(false);
+
+            foreach (var actor in actors)
```
Comment (JesperGomes) [inline source/dotnet/Services/Application/Processes/ProcessApplicationService.cs:89]: What about `foreach (var energySupplier in energySuppliers)` ? Just a suggestion.

**Review by JesperGomes — DISMISSED** [review]

File: source/dotnet/Services/Application/Processes/ProcessApplicationService.cs (hunk 2 above)
Comment (lasrinnil) [reply source/dotnet/Services/Application/Processes/ProcessApplicationService.cs:89]: That's more explicit! Consider it done! :)

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**